data/base/registration/sle15/packages.yaml

```diff
@@ -0,0 +1,15 @@
+packages:
+  bootstrap:
+    on-demand:
+      - cloud-regionsrv-client
+      - regionsrv-certs
+      - sle-module-basesystem-release
+      - sle-module-cap-tools-release
+      - sle-module-containers-release
+      - sle-module-desktop-applications-release
+      - sle-module-development-tools-release
+      - sle-module-legacy-release
+      - sle-module-public-cloud-release
+      - sle-module-python2-release
+      - sle-module-server-applications-release
+      - sle-module-web-scripting-release
```

Comment:
Naming is probably not idea, agreed. The packages in registration are only in on-demand images not in byos images, but they're not csp specific that's why included them in the base branch rather than csp.

Reply:
No problem with the location in the base, this is a slicing topic in that the modules get added to on-demand. So yes the naming/slicing needs to be fixed.

Reply:
I renamed it to `cloudreg` to make it somewhat clearer that this is for cloudregister functionality not for wider SLES registration.

Reply:
We still have the -release packages in this snippet. But release packages get installed in `bootsrap` to avoid IBS related unpleasantness with -mini packages and the actual registration code is part of `images`. While registration is tangentially related to the `-release` packages because of the implementation of `cloud-regionsrv-client` I still don't think that keg-recipes should have "knowledge" of this relationship. I think from a keg-recipes perspective we should break the `-release` packages into their own snippet.

But I do not think this is a blocker for this initial PR.

Reply:
Just noticed, `cloud-regionsrv-client` and `regionsrv-certs` should not be in `bootstrap` but `image`. I'll fix it. I've put those together with the release files because they end up together in the images, so I didn't think further slicing would be necessary, but of course possible if a more logical separation is desired. Since I have to touch it anyway I can move them into separate snippets if you prefer.

Reply:
Yes please. That way we can clearly show the difference between on-demand and byos in the `-release` packages we  add and keep it separated from the registration.

Reply:
I've split it into `cloudreg` and `sle-modules`.